api/gateway/apimiddleware/process_request.go

```diff
@@ -116,7 +116,11 @@ func HandleGrpcResponseError(errJson ErrorJson, resp *http.Response, respBody []
 		// Something went wrong, but the request completed, meaning we can write headers and the error message.
 		for h, vs := range resp.Header {
 			for _, v := range vs {
-				w.Header().Set(h, v)
+				if strings.HasSuffix(h, "Eth-Consensus-Version") {
```

Comment:
Let's use the variable instead of hardcoding the string here and also below

Reply:
maybe I should just extract the constants as header info somewhere else